SparkPost/2web2ui
# AC-1280: Updated collect to use /billing/collect

### What Changed
 - Endpoint is changed to to use `/billing/collect` instead of `/account/billing/collect` 

### How To Test
 - Create an account with a card attached (paid)
 - Change payment info with new card
 - Verify that endpoint called in `/billing/collect` instead of `/account/billing/collect`
